src/main.rs

```diff
@@ -20,6 +21,8 @@ use tracing_subscriber::layer::SubscriberExt;
 use tracing_subscriber::EnvFilter;
 
 const VERSION: &str = env!("CARGO_PKG_VERSION");
+const HTTP_TIMEOUT_SEC: u64 = 5 * 60;
```

Comment:
Maybe `IMDS_HTTP_TIMEOUT_SEC` since we'll add a different one for WireServer?

Reply:
Added prefix `IMDS_`.